.github/workflows/ci.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: '1.22.1'
```

Comment:
k/k is using go1.22 on 1.30 release branch: https://github.com/kubernetes/kubernetes/blob/release-1.30/go.mod - Do we need to use 1.24? So far I think CA was kept in sync with k/k.